guides/common/modules/installing_sos.adoc

```diff
@@ -2,6 +2,10 @@
 
 = Installing the SOS Package on the Host Operating System
 
+ifeval::["{Build}" == "foreman"]
+For Red Hat systems only.
+endif::[]
```

Comment:
Actually Debian contains sosreport as well. I'd probably remove this note but remove references to Red Hat Technical Support and replace this with our community site.

Reply:
I will deal with this when talking with Baboon. Ignore this for now.

Reply:
I will handle this in the debian PRs, if that is OK.

Reply:
Sure, to give you some context: we ship tool called `foreman-debug`, a dirty shell script which I wrote. Last year Pavel Moravec rewrote it into an official sosreport plugin and it's now part of sos package, it will be available in Fedoras and RHEL and we would like to sunset our old tool sometime. However Debian is much more slower paced even then RHEL, so we should keep suggesting the old tool for some time specifically for debian users.